src/protagonist/Portal/Pages/Account/SignupFromLink.cshtml

```diff
@@ -66,17 +49,20 @@
                     <label asp-for="Input.Slug">URL name</label>
                     <input asp-for="Input.Slug" class="form-control" aria-describedby="slugHelp" placeholder="my-org">
                     <span asp-validation-for="Input.Slug" class="text-danger"></span>
-                    <small id="slugHelp" class="form-text text-muted">Your IIIF Image URLs will include this as part of the address.
-                        For example, dlcs.io/iiif-img/<b>my-org</b>/3/my-deep-doom-image.</small>
+                    <small id="slugHelp" class="form-text text-muted">
+                        Your IIIF Image URLs will include this as part of the address.
+                        For example, dlcs.io/iiif-img/<b>my-org</b>/3/my-deep-doom-image.
```

Comment:
I know this was here before but is this supposed to be `my-deep-zoom-image`

Reply:
Turns out it was - fixed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)